xero-payroll-au.yaml

```diff
@@ -4979,14 +5003,47 @@ components:
       enum:
       - REGULATED  
       - SMSF
+    EmploymentType: 
+      type: string
+      enum:
+      - EMPLOYEE  
+      - CONTRACTOR
+    IncomeType: 
+      type: string
+      enum:
+      - SALARYANDWAGES  
+      - WORKINGHOLIDAYMAKER
+      - NONEMPLOYEE  
+      - CLOSELYHELDPAYEES
+      - LABOURHIRE  
+    TaxScaleType: 
+      type: string
+      enum:
+      - REGULAR  
+      - ACTORSARTISTSENTERTAINERS
+      - HORTICULTURISTORSHEARER  
+      - SENIORORPENSIONER
+      - WORKINGHOLIDAYMAKER  
+      - FOREIGN
+    WorkCondition: 
+      type: string
+      enum:
+      - PROMOTIONAL  
+      - THREELESSPERFORMANCESPERWEEK
```

Comment:
The [documentation](https://developer.xero.com/documentation/api/payrollau/types-and-codes/#employees) shows WorkCondition has a 3rd possible value of "NONE"

Reply:
Thanks, good catch!

Reply:
Resolved ✔️